packages/platform/atoms/src/lib/filterAndFormatTimezones.ts

```diff
@@ -0,0 +1,20 @@
+export const filterAndFormatTimezones = (
+  availableTimezones?: { city: string; timezone: string }[],
+  timezonesFromProps?: string[]
+) => {
+  if (!Array.isArray(availableTimezones)) return [];
```

Comment:
If would write
```
if (!availableTimezones) return []
```

and same for check
```
if (timezonesFromProps?.length > 0) {
```

Also, if we pass them from props we don't need to wait for `availableTimezones` so we need to move returning timezones from props to top because we don't need to wait until we fetch timezones if we have props.